justfile

```diff
@@ -0,0 +1,52 @@
+pull-request: fmt prqlc-test
+
+fmt:
+    @ echo '--- remove trailing whitespace ---'
+    @ rg '\s+$' --files-with-matches --glob '!*.rs' . \
+        | xargs -I _ sh -c "echo _ && sd '[\t ]+$' '' _"
+
+    @  echo '--- no-dbg ---'
+    @! rg 'dbg!' --glob '*.rs' . --no-heading
+
+    @ echo '--- cargo fmt ---'
+    @ cargo fmt --all
+
+    @ echo '--- prettier ---'
+    @ prettier --write . \
+            --config=.prettierrc.yaml \
+            --ignore-path=.prettierignore \
+            --ignore-unknown \
+            --log-level=warn
+
+
+prqlc-lint:
+    @echo '--- clippy ---'
+    @cargo clippy --all --fix --allow-staged
+
+
+target := 'x86_64-unknown-linux-gnu'
```

Comment:
Yeah, as an example — this is coded up to be really flexible in the Taskfile, whereas this will only work on linux

Reply:
I was translating this from GH actions and provided the default for target here.